image_width: 1280
image_height: 960
camera_name: left
camera_matrix:
  rows: 3
  cols: 3
  data: [836.3811461119294, 0, 692.0102704877412, 0, 835.2468991096201, 517.572707776734, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.3837052906105479, 0.1187818836949984, -0.0003722153967523676, -0.001424940886785232, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [0.9995782073541606, 0.003360096088751993, 0.02884644062821744, -0.003410460389259142, 0.999992744580769, 0.001696922436036535, -0.02884052951275712, -0.001794586329766101, 0.999582414469827]
projection_matrix:
  rows: 3
  cols: 4
  data: [774.3112771918526, 0, 659.1462936401367, 0, 0, 774.3112771918526, 527.8366203308105, 0, 0, 0, 1, 0]